rp2040-hal/src/multicore.rs

```diff
@@ -107,6 +107,86 @@ impl<const SIZE: usize> Stack<SIZE> {
     }
 }
 
+/// Entry for a Vector in the Interrupt Vector Table.
+///
+/// Each entry in the Vector table is a union with usize to allow it to be 0 initialized via const initializer
+/// This union also allows for updating the stack pointer or
```

Comment:
or?

Reply:
Gosh, I have no idea what I was thinking at this point.
I'll remove that line, hopefully the train of thought returns when I get a chance to make progress on this code next.